### PR TITLE
Bump the music versions

### PR DESCRIPTION
**Pull request by weeryan17**
# Pull request

 - [x] I have read and agreed to the [code of conduct](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CONTRIBUTING.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [GitHub projects](https://github.com/orgs/CascadeBot/projects).
 - [x] I have tested all of my changes.
 
## Added/Changed
Bump the music versions in an effort to fix some issues. No idea if this will.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,8 +11,8 @@
     <properties>
         <build.number>DEV</build.number>
         <jda.version>4.0.0_39</jda.version>
-        <lavaplayer.version>1.3.47</lavaplayer.version>
-        <lavalink.client.version>f9ab37f</lavalink.client.version>
+        <lavaplayer.version>1.3.50</lavaplayer.version>
+        <lavalink.client.version>4.0</lavalink.client.version>
```
Comment (binaryoverload) [inline pom.xml:15]: This isn't the latest version of Lavalink, this build is from Aug 2019.

`41e1025` is the latest

**Comment by JamesHawkinss** [comment]
you forgot the semicolon

File: pom.xml (hunk 1 above)
Comment (weeryan17) [reply pom.xml:15]: It the latest tag. do we want to update to the latest commit?

**Comment by githubfan** [comment]
merge tingz

**Pull request merged**